libs/perun/components/src/lib/entity-search-select/entity-search-select.component.html

```diff
@@ -1,4 +1,4 @@
-<mat-form-field class="w-100">
+<mat-form-field class="{{ fullWidth ? 'w-100' : '' }}">
```

Comment:
I understand why we would like this change but we can achieve this in little bit more elegant way. Keep the class here as before, but in the parent component put the the search select inside a container of fixed width (e.g. 450px). The search select will take up 100% of it's parent (here by parent I mean element like div). This way we don't need another input parameter.